Gruntfile.js

```diff
@@ -105,7 +84,7 @@ module.exports = function(grunt) {
   });
 
   grunt.registerTask('start', function() {
-    global._BOT = spawn('node', ['build/index'], {stdio: 'inherit'});
+    global._BOT = spawn(babelNode, ['src/index'], {stdio: 'inherit'});
```

Comment:
You might also be able to do `spawn('node', ['--require', 'babel-register', 'src/index'], {stdio: 'inherit'});`